getting-started/GettingStarted.md

```diff
@@ -0,0 +1,48 @@
+
+### Pre-requisites
+You must meet the following criteria to be able to participate in the public preview:
+* The Kubernetes application should have a Helm-chart based application that is free of malware. We highly recommend doing a [pre-scan and fixing the vulnerabilities](https://github.com/bobmital/kubernetes-offer-samples/blob/05611a5c6485db980fe27fb8999af728382b547d/getting-started/Vulnerability-Scan.md) before you push the application through the pipeline.
+* You should have an active publishing tenant or access to a [Partner Center account](https://learn.microsoft.com/en-us/partner-center/overview).
+* You should be able to adhere to several of our supported [billing models (licensing options)](https://learn.microsoft.com/en-us/partner-center/marketplace/marketplace-containers). 
+* Must be actively engaged.
+
+### ISV expectations
+If the necessary steps are followed, you can publish the application within 1-2 weeks! We expect you to be actively engaged and contact us if you face any necessary roadblocks after following the required steps. We appreciate your participation in our program. However, if we notice that your engagement levels have decreased, we may need to give priority to other participants due to the high volume of interest we receive.
+
+### Training
+The Following training materials will help accelerate your onboarding onto the Azure Marketplace.
+* [Mastering Container offers for Kubernetes Apps](https://microsoft.github.io/Mastering-the-Marketplace/container/)
+* [Azure K8s Apps Marketplace Demo Walkthrough](https://www.youtube.com/watch?v=_6yGXUND43s)
+
+
+### Steps to follow
+#### Step 1: Package your application
+This step allows you to package your helm chart-based application and other artifacts such as Azure Resource Manager (ARM) template, manifest file, Helm chart and createUIDef into a Cloud Native Application Bundles (CNAB) package. Please refer to the links below:
+* [Prepare your Kubernetes Apps offer](https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-technical-assets-kubernetes?tabs=windows)
+
+At the end of this step, you will be able to create a CNAB bundle.
+
+#### Step 2: Create the Kubernetes offer
+This step allows you to create an Azure Container offer within Partner Center. You can preview and test the offer before publishing it on the Azure Marketplace.
+* [Create an Azure Container offer](https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-offer-setup)
+
+In order to simplify the process of Vulnerability scanning and certification, Microsoft strongly recommends you follow the steps in the Vulnerability Scan-Microsoft Defender document to understand the vulnerabilities flagged. Please take action to resolve them before pushing the offer through the pipeline.
+* [Vulnerability Scan-Microsoft Defender](https://github.com/bobmital/kubernetes-offer-samples/blob/main/getting-started/Vulnerability-Scan.md)
```

Comment:
Update the link here.

Reply:
Are you saying the vulnerability scan link? I think the link is correct. Can you pls check again?